google/cloud/sql/connector/instance_connection_manager.py

```diff
@@ -299,22 +352,21 @@ async def _perform_refresh(self) -> asyncio.Task:
 
         self._current = self._loop.create_task(self._get_instance_data())
         # Ephemeral certificate expires in 1 hour, so we schedule a refresh to happen in 55 minutes.
-        self._next = self._loop.create_task(self._schedule_refresh(_delay))
+        self._next = self._loop.create_task(self._schedule_refresh())
 
         return self._current
 
-    async def _schedule_refresh(self, delay: int) -> asyncio.Task:
+    async def _schedule_refresh(self) -> asyncio.Task:
```

Comment:
IMO, we should leave this in. Both the Java connector and the Go connector have a way to force a refresh early if something goes wrong, we should consider doing so here.

Reply:
fixed